pkgs/by-name/im/imhex/package.nix

```diff
@@ -1,7 +1,7 @@
 { lib
 , stdenv
 , cmake
-, llvm
+, llvm_17
```

Comment:
should this be changed in all-packages.nix as per https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults ?

Reply:
Seems like it.  Changed!